hawk_eye/data_generation/slice_image.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+""" Contains code to slice up an image into smaller tiles. """
+
+import argparse
+import pathlib
```

Comment:
See how the input filepath strings are manipulated i the argparse here: https://github.com/uavaustin/hawk-eye/blob/master/hawk_eye/inference/find_targets.py